### PR TITLE
Delayed trains positions over event bus bridge

### PR DESCRIPTION
**Pull request by tsegismont**
Initial commit: for now, we publish an event every 3 seconds.

It simulates the polling from the client

Do not merge yet!

### CODE REVIEW

**Comment by tsegismont** [comment]
@galderz @karesti so I got this working. Sharing for early review.

TODO: update the solution code, event bus bridge setup should be an exercise

Ideally, we should update the positions as they are updated by the query listener. WDYT?

**Comment by galderz** [comment]
@tsegismont It looks good. We discussed on Monday the doubts. Shall I integrate or you want to add more things now?

**Comment by tsegismont** [comment]
@galderz I just pushed some changes can you take a look? Right now I am making sure that when deploying with solution profile it works.

**Comment by tsegismont** [comment]
@galderz solution code verified! Go ahead and merge if you don't have any question/concern.

File: workshop-steps/workshop.html [hunk 1]
```diff
@@ -465,6 +465,51 @@
...
+BridgeOptions options = new BridgeOptions();
+// TODO 1 - Modify options to allow sending messages to DELAYED_TRAINS_POSITIONS_ADDRESS to the browser
+sockJSHandler.bridge(options);
+router.route("/eventbus/*").handler(sockJSHandler);</code></pre>
+        <ul>
+          <li><kbd>TODO 1</kbd>
+            <p>Use <kbd>addOutboundPermitted</kbd> method to add a new instance of <kbd>PermittedOptions</kbd><br/>
+              <a class="btn btn-info" role="button" onclick="$('#array-sol-26').toggle();">Show/Hide Solution</a>
+            <div id="array-sol-26" style="display:none;">
+              <pre><code class="java">SockJSHandler sockJSHandler = SockJSHandler.create(vertx);
```
Comment (galderz) [inline workshop-steps/workshop.html]: In the solution, I'd only put the line that they have to write, the rest of the code is the same as given to them.

File: workshop-steps/workshop.html [hunk 2]
```diff
@@ -465,6 +465,51 @@
...
+      }
+    });
+  }</code></pre>
+        <ul>
+          <li><kbd>TODO 2</kbd>
+            <p>Use <kbd>eventBus.publish()</kbd> method to publish the asynchronous result to the
+              <kbd>DELAYED_TRAINS_POSITIONS_ADDRESS</kbd> address<br/>
+              <a class="btn btn-info" role="button" onclick="$('#array-sol-25').toggle();">Show/Hide Solution</a>
+            <div id="array-sol-25" style="display:none;">
+              <pre><code class="java">private void publishPositions() {
```
Comment (galderz) [inline workshop-steps/workshop.html]: Same thing here.

**Review by galderz — APPROVED** [review]

**Comment by galderz** [comment]
@tsegismont Just what we discussed earlier.

**Comment by tsegismont** [comment]
@galderz done, that was coming in a separate commit :-)

**Comment by galderz** [comment]
@tsegismont Cool, so is this ready to go in? Or you want to add/change more things?

**Comment by tsegismont** [comment]
No, it's done. Merging.

**Comment by tsegismont** [comment]
Fixes #4

**Pull request merged**